src/schema/types/queries/Search.ts

```diff
@@ -34,6 +33,25 @@ export const emailSearch = {
   },
 };
 
+export const portfolioSearch = {
+  type: "PortfolioSearchResult",
+  args: {
+    name: nonNull(stringArg()),
+  },
+  resolve: async (_: any, args: any, __: any) => {
+    return await UserService.findUserByName(args.name);
+  },
+};
```

Comment:
타입은 다음과 같이 바꾸고, 이에 맞추어서 resolve 함수도 변경이 필요할 것 같아요. User를 불러오는 것이 아닌, User의 Portfolio를 불러와야 하니까요.
```suggestion
export const portfolioSearch = {
  type: "Portfolio",
  args: {
    name: nonNull(stringArg()),
  },
  resolve: async (_: any, args: any, __: any) => {
    return await UserService.findUserByName(args.name);
  },
};
```